docs/topics/Gateway.md

```diff
@@ -645,7 +656,7 @@ A user's presence is their current state on a guild. This event is sent when a u
 | guild_id | snowflake | id of the guild |
 | status | string | either "idle", "dnd", "online", or "offline" |
 
-### Game Object
+#### Game Object
```

Comment:
Not sure why this was added. If anything, the third `#` should be removed to reflect other object linking conventions of `## Object Name`